boards/catie/zest_core_stm32l4a6rg/sixtron_connector.dtsi

```diff
@@ -52,6 +53,20 @@
                    <DIO19 0 &gpiob 8 0>,
                    <DIO20 0 &gpiob 9 0>;
     };
+
+    sixtron_connector_1_pwms: sixtron-connector-1-pwms {
+    	compatible = "pwm-leds";
+
+    	sixtron_connector_1_pwm1: pwm_dev_1 {
+    		pwms = <&pwm1 3 1000000 PWM_POLARITY_NORMAL>;
+    	};
+    	sixtron_connector_1_pwm2: pwm_dev_2 {
+    		pwms = <&pwm1 2 1000000 PWM_POLARITY_NORMAL>;
+    	};
+    	sixtron_connector_1_pwm3: pwm_dev_3 {
+    		pwms = <&pwm3 3 1000000 PWM_POLARITY_NORMAL>;
```

Comment:
Sub-nodes should be named in kebab case.

I'm also wondering if we couldn't simplify the node names:
```suggestion
    	sixtron_connector_1_pwm1: pwm-1 {
    		pwms = <&pwm1 3 1000000 PWM_POLARITY_NORMAL>;
    	};
    	sixtron_connector_1_pwm2: pwm-2 {
    		pwms = <&pwm1 2 1000000 PWM_POLARITY_NORMAL>;
    	};
    	sixtron_connector_1_pwm3: pwm-3 {
    		pwms = <&pwm3 3 1000000 PWM_POLARITY_NORMAL>;
```